base/frontend/sourcegraph-frontend.Deployment.yaml

```diff
@@ -27,7 +27,9 @@ spec:
         deploy: sourcegraph
     spec:
       containers:
-      - args:
+      - name: frontend
+        image: index.docker.io/sourcegraph/frontend:insiders@sha256:b69a210ec21ca10028937dd404467bc84b8da18b31772705199666bc70bc2647
+        args:
```

Comment:
`-args?`

Reply:
`serve` is the arg here (next line)

Reply:
Ugh sorry, the yaml change threw me I thought it was an illegal key for a second. Realized the whole thing was a map.

Reply:
git be like that sometimes haha